### PR TITLE
Portfolio top header

### PR DESCRIPTION
**Pull request by AshwinCarvalho999**
Added index.html
Added style.css
Added Linters
Created navbar section and aligned with flexbox
Created a Headline section for intro text and a brief message
Added geometry design and used font Poppins
Added gitignore file
Added and Edited Readme file
added MIT

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  left: 284px;
+  top: 19px;
+}
+
+.img-geomery-2 {
+  position: absolute;
+  width: 1440px;
+  height: 1643px;
+  top: 375px;
+}
```
Comment (HAMMAS-SALEEM) [inline style.css:72]: - [ ] Please remove these styling to fix the background image. :+1:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+                    project you need coded, don`t hesitate<br />
+                    and contact me.
+               </p>
+
+               <a href="#" class="btn-voilet">Start collaboration</a>
+          </div>
+
+          <div class="img-geomery-2">
+               <img src="images/image_geometry_2.svg" alt="design-geometry" />
+          </div>
```
Comment (HAMMAS-SALEEM) [inline index.html:49]: - [ ] Please add width: 100%; to this image to follow the figma template :+1:

File: index.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+               <p>
+                    Hello! I am a software developer! I can <br />
+                    help you build a product, feature or <br />
+                    website. Taka a look of my works.<br />
+                    If you like what you see and have a<br />
+                    project you need coded, don`t hesitate<br />
+                    and contact me.
+               </p>
+
+               <a href="#" class="btn-voilet">Start collaboration</a>
```
Comment (HAMMAS-SALEEM) [inline index.html:44]: - [ ] Please change this `a` tag to a `button` tag to implement the following styling shown below image :+1:
fix-button

File: style.css [hunk 4]
```diff
@@ -0,0 +1,72 @@
+:root {
+  --yellow-color: #fec25a;
+  --voilet-color: #1a2236;
+  --light-blue-color: #98dede;
+  --dark-blue-color: #1c528a;
+}
+
+.body {
+  margin: 1;
```
Comment (HAMMAS-SALEEM) [inline style.css:9]: - [ ] Please remove this margin: 1; to follow the figma template. :+1:

File: style.css [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+  letter-spacing: -0.01em;
+}
+
+.menu-icon {
+  position: absolute;
+  width: 20px;
+  height: 2px;
+  left: 284px;
+  top: 19px;
+}
```
Comment (HAMMAS-SALEEM) [inline style.css:65]: - [ ] Please make your styling for the .menu-icon as shown in the image below to fix your nav icon position :+1:
nav-icon-styling

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

### Hey @AshwinCarvalho999 :wave:

### Highlights :1st_place_medal:

:heavy_check_mark: Following Linters 
:heavy_check_mark: Following GitHub Flow


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

# Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+               project you need coded, don`t hesitate<br />
+               and contact me.
+          </div>
+          <div class="section">
+               <a href="#" class="button">Start collaboration</a>
+
+               <div class="img-geomery-2">
+                    <img src="images/image_geometry_2.svg" alt="design-geometry" />
+               </div>
+          </div>
```
Comment (codecaiine) [inline index.html:44]: - [ ] [REQUIRED] Nice job so far, But As the previous reviewer mentioned [here](https://github.com/AshwinCarvalho999/Portfolio-new/pull/1#discussion_r915485959) you need to put this content inside `section` tag not in `header`.  :+1:

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @AshwinCarvalho999 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- Descriptive Pull request :heavy_check_mark: 
- Good use of Gitflow :heavy_check_mark: 
- Commits are meaningful :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AshwinCarvalho999** [comment]
Done.

**Comment by AshwinCarvalho999** [comment]
He told me to make the width of image to 100% and i done it

**Comment by HAMMAS-SALEEM** [comment]
Brother, you have to add that image as a background through the CSS I have mentioned above you can see in the requirements. Please add it through CSS as a background image :+1:

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle:

Your project is complete. There is nothing else to say about your project. :+1:
Go ahead and merge it. :muscle:

## Highlights :1st_place_medal:
:heavy_check_mark: Following LInter
:heavy_check_mark: Following GitHub Flow
:heavy_check_mark: Following Requirements
:heavy_check_mark: Great Validations

### [Optional]

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! clap clap clap

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AshwinCarvalho999** [comment]
Awesome, thank you so much!

**Pull request merged**